test/getCreditScore.test.js

```diff
@@ -1,7 +1,8 @@
 const {expect} = require("chai");
-const {getCreditScore} = require('../src/service/getCreditScore')
+const {getCreditScore, getMeanOfFriendsSocialScore} = require('../src/service/getCreditScore')
 
 describe('test getCreditScore', function () {
+    this.timeout(300000);
```

Comment:
po co to :D ?

Reply:
Mówiłem. żeby w testach nie było problemu z timeoutami. Bo jak był ustawiony na dwie sek to często wypluwał timeout i test failował